src/components/screens/home-screen.tsx

```diff
@@ -17,27 +18,60 @@ const HomeScreen = () => {
   const users = useAtomValue(usersAtom);
   const userProfile = useAtomValue(userProfileAtom);
   const loggedInUser = users.find((user: User) => user.id === userProfile?.id);
-  // TODO: Uncomment this code when the optIn feature is ready
-  // const isOptIn = loggedInUser?.attributes?.severaUserId;
-  // const balanceCard = developerMode && isOptIn ? <BalanceCard /> : null;
-  const balanceCard = developerMode ? <BalanceCard /> : null;
-  const sprintViewCard = developerMode ? <SprintViewCard /> : null;
-  const vacationsCard = developerMode ? <VacationsCard /> : null;
-  const questionairesCard = developerMode ? <QuestionnaireCard /> : null;
-  
+  const hasSeveraUserId = !!loggedInUser?.attributes?.severaUserId;
```

Comment:
Is this working? I think Aleksandra is still implementing the UI, as such you should confirm with here that this is the correct way to access the severaUserId.